chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1864,6 +1893,20 @@ public synchronized void saveDump(RenderContext context, TaskTracker progress) {
     }
   }
 
+  private synchronized boolean loadGrid(RenderContext context, TaskTracker progress) {
```

Comment:
Emitter grid? Just in case more grids are added in the future? Same for the filename.